xml/art_sles_rpi_quick.xml

```diff
@@ -211,12 +211,12 @@
    <itemizedlist>
     <listitem>
      <para>
-      Raspberry Pi 3 Model B or Raspberry Pi 3 Model B+
+      &rpi; 3 Model B or &rpi; 3 Model B+ or 3 Model A+ or &rpi; 4 Model B
```

Comment:
I suggest:

"Raspberry Pi 3 Model A+, B or B+, or Raspberry Pi 4 Model B". 

I find out-of-order model names hard to follow.

Reply:
Sounds okay, but it would really help me if you commented on my cleanly separated commits rather on the overall diff, so that I know where to change things.